core/README.md

```diff
@@ -1,4 +1,17 @@
-core
+# core/libdreamcast
 ===========
 
-All of the interesting bits are here
\ No newline at end of file
+Here lies the core of our codebase. Everything that's OS inspecific rests here.
```

Comment:
Wouldn't this better be handled via doxygen or such? Looks a bit too detailed to be in the readme.

Reply:
Well, we do need to have a manual draft of what the automated documentation tool should produce us